src/signals/incident-management/containers/IncidentDetail/components/MapDetail/index.js

```diff
@@ -6,33 +6,25 @@ import { locationType } from 'shared/types';
 
 import './style.scss';
 
-const MapDetail = ({
-  value, hideAttribution, hideZoomControls, useSmallMarker, zoom,
-}) => {
+const MapDetail = ({ value, smallMarker, zoom, hideAttribution, ...otherProps }) => {
   const location = value && value.geometrie && value.geometrie.coordinates;
   const latlng = location ? { latitude: location[1], longitude: location[0] } : null;
   return (
     <div className="map-detail">
-      {latlng
-        ? (
-          <Map
-            latlng={latlng}
-            hideAttribution={hideAttribution}
-            hideZoomControls={hideZoomControls}
-            useSmallMarker={useSmallMarker}
-            zoom={zoom}
-          />
-        ) : ''}
+      {latlng ? (
+        <Map latlng={latlng} smallMarker={smallMarker} hideAttribution={hideAttribution} zoom={zoom} {...otherProps} />
+      ) : (
+        ''
+      )}
     </div>
   );
 };
 
 MapDetail.propTypes = {
   value: locationType.isRequired,
   hideAttribution: PropTypes.bool,
-  hideZoomControls: PropTypes.bool,
-  useSmallMarker: PropTypes.bool,
-  zoom: PropTypes.string,
+  smallMarker: PropTypes.bool,
```

Comment:
Use a descriptive name for props. In this case you could use `useSmallMarker` or `hasSmallMarker` for instance.

Reply:
refactored, the icon can be passed from the parent for flexibility